sematic/docker_images.py

```diff
@@ -0,0 +1,46 @@
+# Standard Library
+import os
+import pathlib
+
+import __main__
+
+CONTAINER_IMAGE_ENV_VAR = "SEMATIC_CONTAINER_IMAGE"
+
+
+class MissingCloudImage(Exception):
+    pass
+
+
+def has_cloud_image() -> bool:
```

Comment:
Why "cloud" image? Why not "container" image?

Same thing about consistency with the env var.

Reply:
I was just thinking that "container" may feel a bit too "infra-y" for MLEs, but I guess it's fine.